src/components/common/overview/SchemaPanel.tsx

```diff
@@ -143,7 +143,7 @@ const renderColumnName = (
   const isPk = _.includes(table.primaryKey || [], column.name);
   retVal.push(
     <span key="dt" className={classes.columnBox} title={column.datatype}>
-      {column.datatype.substring(0, 1).toUpperCase()}
+      {(column.datatype?.substring(0, 1) || 'N').toUpperCase()}
```

Comment:
Is this `N` the default short display name for a column with no datatype?  This seems counterintuitive to me, I would think this would be a number.

Could we just make it the empty string?  I think that's a more appropriate decoration!

Reply:
<img width="1440" alt="Screen Shot 2022-12-21 at 3 49 43 PM" src="https://user-images.githubusercontent.com/10501914/209000323-5341266d-8ae5-408a-ab66-5c93bb0dd20a.png">
This is what it looks like blank - I know the 'n' is counterintuitive too, but the blank just looks wrong to me!

What would you think of using this then?
https://fontawesome.com/search?q=empty&o=r

An icon for empty?

Reply:
Resolution: We updated this so "unknown datatype" now shows '?'